staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -547,6 +547,14 @@ type KubeletConfiguration struct {
 	// Default: false
 	// +optional
 	ProtectKernelDefaults bool `json:"protectKernelDefaults,omitempty"`
+	// warningInvalidKernelDefaults, if ProtectKernelDefaults is set to true, this flag will be ignored.
+	// It only takes effects if ProtectKernelDefaults is false or not set.
```

Comment:
nit: we encourage to use field name reference as shown in API doc. `ProtectKernelDefaults` -> `protectKernelDefaults`

Reply:
Fixed. Thanks